packages/tradershub/src/features/cfd/constants.tsx

```diff
@@ -19,6 +19,7 @@ import { TMarketTypes, TMT5LandingCompanyName, TPlatforms } from '../../types';
 type TAppContent = {
```

Comment:
rather than creating a new key for an icon with width is it possible to pass a classname to it and from there we can control the width, height and etc.?

Reply:
@aizad-deriv no, it's not possible to control the width by wrapper classname